src/components/layouts/CreateIngredientsList.js

```diff
@@ -158,4 +116,34 @@ export default CreateIngredientsList;
 
 const styles = StyleSheet.create({
```

Comment:
Maybe i was unclear before.
Main idea of this task was to move Stylesheet component to a separated file(aka styles), from components code. Because we did this screens quickly, and it's confusing to have such long code files for such simple views.

Please review articles from the #66 task. tell me if something is not clear